### ISSUE TITLE
[P2][Sourcing][Accessibility][Pretest][Multi input autocomplete] If the input field is moved away with TAB, the autocomplete list receives focus 

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#10815]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
If the input field is moved away with TAB, the autocomplete list receives focus 
expectation: Keyboard focus should be placed at the next control. The autocomplete list should not receive focus with TAB, but only with arrow keys

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
Angular: 15
core/platform: 0.43.14

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
1) Type "App" and the input field has Apple with autocomplete list opened
2) Now press tab , the focus goes to flat list

**Comment by mikerodonnell89** [comment]
fixed by https://github.com/SAP/fundamental-ngx/pull/10865
